example/server.yaml

```diff
@@ -1,11 +1,15 @@
 host: ''
 env: 'production'
 server:
-  name: 'test'
+  name: 'test2'
   instanceType: 't2.nano'
   instanceSize: 8
-  keyName: 'default'
+  keyName: 'default-test'
   users:
-    - 'jgallen23'
+    - 'ecwillis'
+
+ec2:
+  SubnetId: 'subnet-4fefa02a'
```

Comment:
This was for my account which doesn't have default subnet ids for the vpc. There is documentation out there somewhere, but if you don't include this - the command fails.